### PR TITLE
Lms 96

### PR DESCRIPTION
**Pull request by MikolaLvivUA**

### CODE REVIEW

File: src/controllers/lesson/lesson.controller.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+      const lessonValidity = Joi.validate(lesson, lessonValidator);
+
+      if (lessonValidity.error) {
+        return next(new ErrorHandler(ResponseStatusCodesEnum.BAD_REQUEST, lessonValidity.error.details[0].message));
+      }
+
+      await lessonService.createLesson(lesson);
+
+      res.status(ResponseStatusCodesEnum.CREATED);
```
Comment (VictorKmin) [inline src/controllers/lesson/lesson.controller.ts]: It's not will be working without .end

File: src/routes/api/api.router.ts [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
 const router = Router();
 router.use('/auth', authRouter);
 
-router.use(checkAccessTokenMiddleware);
+router.use(checkAccessTokenMiddleware, checkIsAdmin);
 router.use('/users', userRouter);
 router.use('/groups', groupRouter);
 router.use('/courses', courseRouter);
 router.use('/modules', moduleRouter);
+router.use('./lessons', lessonRouter);
```
Comment (VictorKmin) [inline src/routes/api/api.router.ts]: ./lessons ?

File: src/services/course/course.service.ts [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import { model } from 'mongoose';
 
-import { config } from '../../configs';
+import { DatabaseTablesEnum } from '../../constants/enums';
```
Comment (VictorKmin) [inline src/services/course/course.service.ts]: import { DatabaseTablesEnum } from '../../constants';

File: src/services/lesson/lesson.service.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import { Lesson } from '../../database';
+import { ILesson } from '../../interfaces';
+
+class LessonService {
+
+  createLesson(lessonValue: ILesson): Promise<any> {
```
Comment (VictorKmin) [inline src/services/lesson/lesson.service.ts]: createLesson(lessonValue: ILesson): Promise<void> {

File: src/services/user/user.service.ts [hunk 5]
```diff
@@ -1,6 +1,6 @@
 import { model } from 'mongoose';
 import { User, UserSchema, UserType } from '../../database';
-import { IUser } from '../../Interfaces';
+import { IUser } from '../../interfaces';
 
 class UserService {
     createUser(userValue: IUser): Promise<any> {
```
Comment (VictorKmin) [inline src/services/user/user.service.ts]: createUser(userValue: IUser): Promise<void> {

File: src/controllers/lesson/lesson.controller.ts (hunk 1 above)
Comment (MikolaLvivUA) [reply src/controllers/lesson/lesson.controller.ts]: fixed

File: src/routes/api/api.router.ts (hunk 2 above)
Comment (MikolaLvivUA) [reply src/routes/api/api.router.ts]: fixed

File: src/services/course/course.service.ts (hunk 3 above)
Comment (MikolaLvivUA) [reply src/services/course/course.service.ts]: fixed

File: src/services/lesson/lesson.service.ts (hunk 4 above)
Comment (MikolaLvivUA) [reply src/services/lesson/lesson.service.ts]: fixed

File: src/services/user/user.service.ts (hunk 5 above)
Comment (MikolaLvivUA) [reply src/services/user/user.service.ts]: fixed

**Pull request merged**